test-bdd/main_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"io/ioutil"
 	"testing"
 
-	addonmgrv1alpha1 "github.com/keikoproj/addon-manager/api/v1alpha1"
+	addonmgrv1alpha1 "github.com/keikoproj/addon-manager/pkg/apis/addon/v1alpha1"
```

Comment:
Why moving everything to `/pkg` dir, the newer way is to have top-level domains and actually `/pkg` doesn't exist... newest kubebuilder actually no longer vends a `pkg/` dir?

Reply:
Just following controller pattern, put API and Client under pkg for better reference, put both API and Client under root, make people confused.

Reply:
we move API back https://github.com/keikoproj/addon-manager/pull/99/files#diff-b2b641185a128162d70112b66fa467c5f169791ad933222d842ae06656278abfL295